sklearn/linear_model/tests/test_ransac.py

```diff
@@ -22,15 +22,12 @@
 data = np.column_stack([X, y])
 
 # Add some faulty data
-outliers = np.array((10, 30, 200))
-data[outliers[0], :] = (1000, 1000)
-data[outliers[1], :] = (-1000, -1000)
-data[outliers[2], :] = (-100, -50)
-
+np.random.seed(1)
```

Comment:
OK, but it's important to seed with a different seed that I use in the test, or I will "randomly" sample only outliers. That was fun

I used rng = np.random.RandomState(1000)